modules/node-observability-create-custom-resource.adoc

```diff
@@ -10,7 +10,7 @@ Before you run profiling queries, you must create a `NodeObservability` custom r
 
 [IMPORTANT]
 ====
-Creating a `NodeObservability` CR reboots all the worker nodes. It might take 10 or more minutes to complete.
+The worker nodes which are only specified in `NodeObservability` CR are rebooted. It might take 10 or more minutes to complete.
```

Comment:
```suggestion
The nodes which match the nodeSelector specified in `NodeObservability` CR will be rebooted. It might take 10 or more minutes to complete.
```